src/nvim/ex_cmds.h

```diff
@@ -16,7 +16,8 @@
 #define ECMD_OLDBUF          0x04    // use existing buffer if it exists
 #define ECMD_FORCEIT         0x08    // ! used in Ex command
 #define ECMD_ADDBUF          0x10    // don't edit, just add to buffer list
-
+#define ECMD_ALTBUF          0x20    // like ECMD_ADDBUF and set the alternate
```

Comment:
This can be reindented instead so that it fits in 1 line like Vim. Neovim doesn't use tabs and avoids vertical alignment to avoid whitespace changes in some parts of the codebase (ie. src/nvim/eval/ function docstrings).